companion/HELP.md

```diff
@@ -0,0 +1,4 @@
+## PPT VIDEO
+All the presets are done, but if you have any questions, send me a message on Instagram. Monteiro.audiovisual
```

Comment:
The help file should give help about the module, the configuration, the actions, the feedbacks and so on. If I read this, I get no help. And I have questions: what is this connection for? I can't see its purpose.